src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -1080,686 +285,4 @@ static inline int MPIDI_NM_comm_get_lpid(MPIR_Comm * comm_ptr,
     return MPI_SUCCESS;
 }
```

Comment:
Can this function go to some other file that's still inlined given that it's the only function left here? `ofi_impl.h` for instance? If so, we can just get rid of this file.

Reply:
I was trying to do that. Then I realized that this function is used `MPIDIU_valid_group_rank` which is used by `MPIDIG_EPOCH_CHECK_TARGET_SYNC` macro. So I left it in the header for inlining.

Reply:
I agree that it should still be inlined. I'm just saying that if it's the only function left in this file, it's a little silly. It can go in another file that's being inlined (like `ofi_impl.h`).

Reply:
I see. That's a good idea!

Reply:
I ended up moving this function into `ofi_proc.h` because `ofi_impl.h` is included in many `.c` file which in turn will create their own copy of `MPIDI_NM_comm_get_lpid` and conflict with each other.